vpr/src/place/place.cpp

```diff
@@ -447,6 +447,8 @@ void try_place(const Netlist<>& net_list,
      * width of the widest channel.  Place_cost_exp says what exponent the   *
      * width should be taken to when calculating costs.  This allows a       *
      * greater bias for anisotropic architectures.                           */
+
+    VTR_ASSERT(!is_flat);
```

Comment:
Comment why this assert is here.  Should have a TODO as well. Or if we never need flat routing in placement, remove is_flat.